docker-compose.yml

```diff
@@ -363,6 +363,9 @@ services:
   transactions-consumer:
     <<: *sentry_defaults
     command: run consumer ingest-transactions --consumer-group ingest-consumer
+  ingest-feedback-events:
+    <<: *sentry_defaults
+    command: run consumer ingest-feedback-events --consumer-group ingest-consumer
```

Comment:
Should we use `ingest-feedback-events` as the consumer group instead?

Reply:
Does this consumer group exist for self-hosted? I was following the convention of the other ingest consumers in this file

Reply:
Will the command create the group?

Reply:
Other ingest consumers in this file also have different consumer groups. The reason why transactions, attachments, and events all have the same consumer group is because they used to be combined, but were split in https://github.com/getsentry/self-hosted/pull/2193/files